python/symbiflow_test.py

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python3
```

Comment:
```
#
```

Reply:
Made the license header look exactly as we have in prjxray